solr/core/src/test/org/apache/solr/cloud/DistributedVersionInfoTest.java

```diff
@@ -58,7 +56,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-@LuceneTestCase.Nightly // Lots of sleeps to introduce timing delays?
+// @LuceneTestCase.Nightly // Lots of sleeps to introduce timing delays?
```

Comment:
Why?

Reply:
argh, this was so I could test this test...   I need to learn how to pass parameters to tests so I can run nightly!  good catch...